angularjs-portal-home/src/main/webapp/css/variables.less

```diff
@@ -22,6 +22,9 @@
    ========================================================================== */
 @white: #FFFFFF;
 @black: #000000;
+@primary: #c5050c;
+@primaryDarkened20: #920000;
+@accent: #066999;
```

Comment:
Please keep in mind theme support for other schools when making these color choices. See css/theme directory.